integration-tests/requirements.txt

```diff
@@ -1,3 +1,5 @@
+urllib3<2
+cython<3
```

Comment:
I wasn't sure, investigated and it was due to an issue with PyYAML that is used in the docker requirement, which is incompatible (see https://github.com/yaml/pyyaml/issues/601 ).  Can be fixed too with a different pyyaml version.

I've removed that now and it seems to craete the venv ok, except I get an error using confluent_kafka == 1.9.2 now?  Can you check?  Will clear my environment and check again tomorrow with a clear head.

Reply:
Ahh the confluent_kafka issue is because the conan package (which also has version 1.9.2) wasn't in my path and being picked up by Python and instead it was using the latest version I had installed via macports.

Reply:
happy to resolve and approve?  Or prefer without the python restriction?